uint8   obj_id
uint8   sub_id
float32 accuracy
run_task/ArmPoseWithID[] arm_poses 